recipes/ipyreact/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ipyreact" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ipyreact-{{ version }}.tar.gz
+  sha256: 968e40000e6792a62c63aeb64ef897f1ae569696c6df71c939593554c1a51de9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling >=1.3.1
+    - hatch-jupyter-builder >=0.3.2
+    - jupyterlab ==3.*
+    - pip
+  run:
+    - python >=3.7
+    - ipywidgets >=7.0.0
+    - anywidget >=0.2.0
+
+test:
+  imports:
+    - ipyreact
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/widgetti/ipyreact
+  summary: React for ipywidgets that just works
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - maartenbreddels
+    - kolibril13
```

Comment:
cc @maartenbreddels @kolibril13 do you agree to be listed as maintainers here?

Reply:
Sure thanks!

Reply:
I guess I'm forced now 😄